ui/src/app/pages/admin/user/admin-user-bypass.tsx

```diff
@@ -78,16 +78,20 @@ export class AdminUserBypass extends React.Component<Props, State> {
 
   async save() {
     const { selectedModules } = this.state;
-    const { user } = this.props;
+    const {
+      user,
+      user: { username },
+    } = this.props;
     const changedModules = fp.xor(getBypassedModules(user), selectedModules);
     this.setState({ isSaving: true });
     try {
-      for (const module of changedModules) {
-        await userAdminApi().bypassAccessRequirement(user.userId, {
-          isBypassed: selectedModules.includes(module),
-          moduleName: module,
-        });
-      }
+      await userAdminApi().updateAccountProperties({
```

Comment:
benefits of this change
* 1 call instead of N
* uses the same endpoint that the individual-user admin does

Reply:
What do you mean by "individual-user admin"?

Reply:
I'm referring to a different Admin UI pages.

There's an all-users admin table page `admin-user-table` which contains this `admin-user-bypass` component, and also links to pages for individual users `admin-user-profile`.  These pages also allow admin features like bypass.

`admin-user-profile` uses `updateAccountProperties()`, so now `admin-user-bypass` matches this behavior.